portfolio/src/main/webapp/commentform.html

```diff
@@ -9,18 +9,30 @@
     <!-- using Google's CDN library for jQuery -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
   </head>
-  <body class = "main-body" onload = "loadComments()">
+  <body class = "main-body" onload = "loadAllComments()">
         <div id = "nav"> </div>
         <script> $(document).ready(function(){$("#nav").load("navbar.html")}); </script>
         <form action = "/data" method = "POST">
-            <p id = "commentText"> Comment: </p>
+            <p class = "commentText"> Enter a number between 0 and 20 for the number of comments 
+                you would like to be displayed: </p>
+            <input type = "number" name = "numComments" id = "max" min = "0" max = "20" value = "1" >
+            <br>
```

Comment:
nit: looks like there is mixed usage of <br> and <br/> here. Try to use <br> everywhere